x/evmutil/types/msg.go

```diff
@@ -144,3 +154,94 @@ func (msg MsgConvertERC20ToCoin) Route() string {
 func (msg MsgConvertERC20ToCoin) Type() string {
 	return TypeMsgConvertERC20ToCoin
 }
+
+// Route implements Msg
+func (msg MsgEVMCall) Route() string { return types.RouterKey }
+
+// Type implements Msg
+func (msg MsgEVMCall) Type() string { return sdk.MsgTypeURL(&msg) }
+
+// ValidateBasic implements Msg
+func (msg MsgEVMCall) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(msg.Authority); err != nil {
```

Comment:
here we could validate the the authority address is in the whitelist of addresses

Reply:
this is already validated in `msg_sever`. Basic validation cannot get data from state so its not possible to validate whitelists here.